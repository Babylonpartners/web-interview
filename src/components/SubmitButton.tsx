import * as React from 'react'
import { activeButtonStyles } from '../styles/styles';

export class SubmitButton extends React.PureComponent {
    render() {
        return (

            <>
            <hr />
            <br />
            <div>
              <button
                style={activeButtonStyles}
                data-testid="submitButton"  
                className="submit-button"
                type="submit"
              >
                Book
              </button>
              <br />
              <br />
              <br />
            </div>
            </>

        );
}}