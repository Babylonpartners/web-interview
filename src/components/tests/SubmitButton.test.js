import React from 'react'
import { render, cleanup, fireEvent } from '@testing-library/react'
import { SubmitButton } from '../SubmitButton'

afterEach(cleanup)

test('<SubmitButton>', () => {
  const { getByTestId, container } = render(<SubmitButton />)
  expect(container.firstChild).toMatchSnapshot()
  expect(getByTestId('submitButton')).toBeTruthy()
})
