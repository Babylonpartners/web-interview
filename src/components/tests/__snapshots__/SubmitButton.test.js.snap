// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubmitButton> 1`] = `<hr />`;
